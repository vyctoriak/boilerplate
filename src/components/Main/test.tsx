import { render, screen } from '@testing-library/react'
import Main from '.'

describe('<Main />', () => {
  it('should render the heading', () => {
    // render the component
    const { container } = render(<Main />)
    // seach for the element
    expect(
      screen.getByRole('heading', { name: /react avan√ßado/i })
    ).toBeInTheDocument()
    // verify if exists

    // generate snapshot
    expect(container.firstChild).toMatchSnapshot()
  })
})
