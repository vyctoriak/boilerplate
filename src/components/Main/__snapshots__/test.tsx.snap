// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> should render the heading 1`] = `
<div
  class="sc-bcPKhP jhrJfq"
>
  <h1>
    React Avan√ßado
  </h1>
</div>
`;
